python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py

```diff
@@ -191,7 +191,7 @@
                  '$x_2$', '$Q_1(x_2)$', True, '')
```

Comment:
I think the title of the last graph is also wrong: it should be X2 | X1=x1

Reply:
Ok. I change it.